style.css

```diff
@@ -66,10 +66,9 @@
 }
 
 .box {
-  border: #d3d3d3 solid 3px;
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/198573722-84c39e33-7998-4272-81b5-73d379adc013.PNG)

**[Optional]**

Kindly make your design responsible for big screens as well means resolution more than 1400px screen size to make your project more professional and to catch the recruiters eyes by the responsiveness of your application 👍